src/olympia/activity/utils.py

```diff
@@ -136,8 +138,21 @@ def log_and_notify(action, comments, note_creator, version):
         alog.user for alog in ActivityLog.objects.for_version(version)]
     # Collect add-on authors (excl. the person who sent the email.)
     addon_authors = set(version.addon.authors.all()) - {note_creator}
-    # Collect reviewers on the thread (again, excl. the email sender)
-    reviewer_recipients = set(log_users) - addon_authors - {note_creator}
+    # Collect staff that want a copy of the email
+    try:
+        cc_group = models.Group.objects.get(name='Activity Mail CC')
+        staff_cc = [user for user in cc_group.users.all()]
```

Comment:
Hmm.  I had it as a single line of `staff_cc = [user for user in models.Group.objects.get(name='Activity Mail CC').users.all()]` but line wrapping made it hard to follow btw.

Regarding making it more efficient - do the relative counts make a difference to filtering via User->GroupUser->Group rather than queries of Group then GroupUser->User?  User is ~5.2 million; GroupUser likely a few hundred; Group 70.

As I've been banging on about using constants instead of names we probably should, heh.  Though I'd note it's only in exceptional circumstances we should be looking up Groups, or checking for group membership, because in 99% of cases it's the permission a known user(s) has we are interested in.  In this case searching through said 5.2 million users to find ones that indirectly have a permission because they are members of a group that has matching rules, including *, is even more crazy though.